pulp_ansible/app/serializers.py

```diff
@@ -1,14 +1,18 @@
 from gettext import gettext as _
 
+from django.db import transaction
 from django.conf import settings
 from jsonschema import Draft7Validator
 from rest_framework import serializers
+from urllib.parse import urljoin
 
-from pulpcore.plugin.models import Artifact, SigningService
+from galaxy_importer.constants import NAME_REGEXP
+from pulpcore.plugin.models import Artifact, ContentArtifact, SigningService
 from pulpcore.plugin.serializers import (
     DetailRelatedField,
     ContentChecksumSerializer,
     ModelSerializer,
+    NoArtifactContentSerializer,
```

Comment:
I don't think that this was caused by this pr, but `TaskSerializer` isn't available in pulpcore 2.21. Someone needs to update the minimum supported version for pulp_ansble, and or backport `TaskSerializer`